protocols/migration/participants.py

```diff
@@ -583,3 +584,158 @@ def migrate_tumour_sample(self, tumour_sample):
         return self.validate_object(
             object_to_validate=migrated_tumour_sample, object_type=self.new_model.TumourSample
         )
+
+
+class MigrationParticipants100ToReports(BaseMigration):
+    old_model = participant_1_0_0
+    new_model = participant_old
+
+    def migrate_pedigree(self, old_pedigree):
+        """
+        :param old_pedigree: org.gel.models.participant.avro.Pedigree 1.0.0
+        :return: org.gel.models.report.avro RDParticipant.Pedigree 3.0.0
+        """
+        new_pedigree = self.convert_class(self.new_model.Pedigree, old_pedigree)
+        new_pedigree.versionControl = self.new_model.VersionControl()
+        new_pedigree.gelFamilyId = old_pedigree.familyId
+        new_pedigree.analysisPanels = self.migrate_analysis_panels(old_panels=old_pedigree.analysisPanels)
+        new_pedigree.participants = self.migrate_members_to_participants(old_members=old_pedigree.members,
+                                                                         family_id=old_pedigree.familyId)
+        return self.validate_object(object_to_validate=new_pedigree, object_type=self.new_model.Pedigree)
+
+    def migrate_members_to_participants(self, old_members, family_id):
+        return [self.migrate_member_to_participant(old_member=old_member, family_id=family_id) for old_member in
+                old_members]
+
+    def migrate_member_to_participant(self, old_member, family_id):
+        new_participant = self.convert_class(self.new_model.RDParticipant, old_member)
+        new_participant.gelFamilyId = family_id
+        new_participant.pedigreeId = old_member.pedigreeId
```

Comment:
`old_member.pedigreeId` can be None, but it is a non nullable field in report 3.0.0. Should be set to `0` if `None`